docs/requirements.txt

```
@@ -1,4 +1,4 @@
-sphinx >=4.4
+sphinx ==7.1.2
```

to be consistent with rascaline

```suggestion
sphinx < 7.2    # breathe is not compatible with sphinx 7.2 yet
```

---

I tried this first, but it gives me error with furo (it seems to resolve to sphinx 6.2 which also fails)

---

not sure why rascaline is not failing ...

---

okay very nice! lets go for it.